packages/compass-e2e-tests/helpers/commands/create-search-index.ts

```diff
@@ -0,0 +1,39 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function createSearchIndex(
+  browser: CompassBrowser,
+  indexName: string,
+  indexDefinition: string,
+  screenshotName?: string
+): Promise<void> {
+  await browser.waitUntil(async () => {
+    await browser.clickVisible(Selectors.CreateIndexDropdownButton);
```

Comment:
You probably want to move this outside of the wait. And then you can also just:

`await browser.$(Selectors.createIndexDropdownAction('search-indexes')).waitForDisplayed();`

which is builtin and I think a bit more idiomatic.